src/DataTypes/Serializations/SerializationAggregateFunction.cpp

```diff
@@ -215,4 +215,17 @@ void SerializationAggregateFunction::deserializeTextCSV(IColumn & column, ReadBu
     deserializeFromString(function, column, s, version);
 }
 
+void SerializationAggregateFunction::serializeTextMarkdown(
+    const DB::IColumn & column, size_t row_num, DB::WriteBuffer & ostr, const DB::FormatSettings & settings) const
+{
+    if (settings.output_format_markdown_escape_special_characters)
+    {
+        writeMarkdownEscapedString(serializeToString(function, column, row_num, version), ostr);
```

Comment:
Serialized aggregate function states are afaik binary. I guess we can always use the default implementation (`serializeTextEscaped()`).